src/manifest.json

```diff
@@ -13,10 +13,10 @@
       "sizes": "512x512"
     }
   ],
-  "start_url": "/",
+  "start_url": ".",
   "background_color": "#3367D6",
   "display": "standalone",
-  "scope": "/",
+  "scope": ".",
```

Comment:
nit: both `start_url` and `scope` are optional, we can probably remove them altogether?

Reply:
done